<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>941</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>911</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>添加物理块</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>删除物理块</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fifoBtn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>FIFO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lruBtn">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>30</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>LRU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>731</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入页面引用序列，以空格间隔</string>
   </property>
  </widget>
  <widget class="QLabel" name="lackTime">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>940</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lackRate">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>940</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>920</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="lackTime_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>940</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>缺页次数</string>
   </property>
  </widget>
  <widget class="QLabel" name="lackRate_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>940</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>缺页率</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>800</y>
     <width>941</width>
     <height>101</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>淘汰页号</string>
    </property>
   </row>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>670</y>
     <width>941</width>
     <height>111</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>是否缺页</string>
    </property>
   </row>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1040</x>
     <y>830</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
